Extensions/TweenBehavior/JsExtension.js

```diff
@@ -855,9 +855,9 @@ module.exports = {
       .addParameter('expression', _('To Hue'), '', false)
       .addParameter('yesorno', _('Animate Hue'), '', false)
       .setDefaultValue('yes')
-      .addParameter('expression', _('To Saturation (-1 to ignore)'), '', false)
+      .addParameter('expression', _('To Saturation (-1 to ignore to 100)'), '', false)
       .setDefaultValue('-1')
-      .addParameter('expression', _('To Lightness (-1 to ignore)'), '', false)
+      .addParameter('expression', _('To Lightness (-1 to ignore to 100)'), '', false)
```

Comment:
What do you think about `(0 to 100, -1 to ignore)`?

Reply:
I agree 👍